apiary.apib

```diff
@@ -1688,6 +1688,103 @@ For example you should create a group for the Introduction to Programming course
             }
         }
 
+## Group [/v1/issuer/groups/{group_id}/clone]
+
+### Clone a Group [POST]
+You can clone a group using this action. Add attributes to override values.
```

Comment:
I think it would be better if we add a note about how the name of new group would be if none if provided.

Reply:
+1 for adding more explanation around the behavior.